bin/cli.js

```diff
@@ -48,6 +38,33 @@ if (program.config) {
 
 var output = config.output || program.output
 
+var args = program.args || []
+var globs
+
+// if config has an input parameter, try to use it
+if (config.input) {
+  if (!Array.isArray(config.input)) {
+    console.log('  [error] '.red + '`input` must be an array when specified in the config')
+    program.help()
+    program.exit(1)
+  }
+
+  globs = config.input.map(function (s) {
+    return path.resolve(path.dirname(path.resolve(program.config)), s)
+  })
+}
+
+// otherwise, expect to have it specified in the cli
+else {
+  var globs = args.map(function (s) {
+    s = s.trim()
+    if (s.match(/(^'.*'$|^".*"$)/)) {
+      s = s.slice(1, -1)
+    }
+    return s
+  })
+}
```

Comment:
This prefers config value over values in the command line. Should it be the other way? Should it merge them?

Reply:
I think it is more common for the cli arguments to have precedence over the config file. Would you mind switching the logic?

Reply:
Not at all. Switched!